release/pipeline.yaml

```diff
@@ -0,0 +1,347 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: release
```

Comment:
I would use something more descriptive, `full-release-candidate-testing` maybe (feel free to play with AI to get better name)?
I might even consider renaming this file to `full-release-candidate-testing-pipeline.yaml` maybe.

Reply:
I'd refactor naming in another PR for the following issue https://github.com/Kuadrant/testsuite-pipelines/issues/100